src/components/bottomSheetBackdrop/BottomSheetBackdrop.tsx

```diff
@@ -119,12 +128,11 @@ const BottomSheetBackdropComponent = ({
       <Animated.View
         style={containerStyle}
         pointerEvents={pointerEvents}
-        accessible={true}
-        accessibilityRole="button"
-        accessibilityLabel="Bottom Sheet backdrop"
-        accessibilityHint={`Tap to ${
-          typeof pressBehavior === 'string' ? pressBehavior : 'move'
-        } the Bottom Sheet`}
+        accessible={_providedAccessible ?? undefined}
+        accessibilityRole={_providedAccessibilityRole ?? undefined}
+        accessibilityLabel={_providedAccessibilityLabel ?? undefined}
+        accessibilityHint={_providedAccessiblityHint ?? undefined}
```

Comment:
@Mahmoud-SK can we maybe do this in order to not break the existing behaviour?
```
accessibilityHint ={_providedAccessiblityHint  ? _providedAccessiblityHint  : `Tap to ${
          typeof pressBehavior === 'string' ? pressBehavior : 'move'
        } the Bottom Sheet`}
```

Reply:
Done. :)